geode-docs/developing/query_index/creating_map_indexes.html.md.erb

```diff
@@ -58,4 +58,7 @@ In order to create or query a map index, you must use the bracket notation to li
 **Note:**
 You can still query against Map or HashMap fields without querying against a map index. For example, you can always create a regular range query on a single key in any Map or HashMap field. However, note that subsequent query lookups will be limited to a single key.
 
+**Note:**
+Map indexes on all keys (when `*` is specified) are not used in queries where the map field is compared with `null` or when it is compared using `!=`, e.g. `p.positions['company1'] = null` or `p.positions['company1'] != '3'`.
+These types of queries will be executed without making use of the index which could make them slower.
```

Comment:
Please insert a comma after "index":  ...making use of the index, which could...